dashboards/ec2/ec2_application_load_balancer_detail.sp

```diff
@@ -111,9 +134,9 @@ query "aws_alb_graph_relationships" {
         'DNS Name', alb.dns_name
       ) as properties
     from
-      alb
+      alb 	
 
-    -- security groups - nodes
+    -- To security group (node)
```

Comment:
```suggestion
    -- To VPC security groups (node)
```

Please see formatting notes for the comment as per https://github.com/turbot/steampipe-utils/wiki/Resource-Relationship-Graph-Checklist#nodes